packages/components/pie-notification/src/index.ts

```diff
@@ -48,6 +49,10 @@ export class PieNotification extends LitElement implements NotificationProps {
     @validPropertyValues(componentSelector, variants, 'neutral')
     public variant: NonNullable<NotificationProps['variant']> = 'neutral';
 
+    @property()
+    @validPropertyValues(componentSelector, positions, 'inline-content')
+    public position: NonNullable<NotificationProps['position']> = 'inline-content';
```

Comment:
I am interested in this pattern of using NonNullable. Could you explain a bit more about how it works please @thejfreitas ? :)

Reply:
To make it explicit that in the component's class scope this property is not nullable and all internal methods will assume that this prop will have a value regardless if is optional for our users.